.github/workflows/release.yml

```
@@ -0,0 +1,126 @@
+          release-type: simple
+          package-name: cloud-sql-java-connector
+  
+  update-pom-files:
+    runs-on: ubuntu-latest
+    needs: release-please
+    if: ${{ needs.release-please.outputs.pr }}
+    steps:
+      - name: update pom.xml files
+        run: ./.release/update_versions.sh
+      - name: push commit
+        uses: stefanzweifel/git-auto-commit-action@v4
+        with:
+          commit_message: "update versions in pom.xml files and READMEs"
+          ref: ${{ fromJson(needs.release-please.outputs.pr).headBranchName }}   
+
+  upload_uberjars:
+    runs-on: ubuntu-latest
+    needs: release-please
+    if: ${{ needs.release-please.outputs.releases_created }}
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - id: 'auth'
+      name: 'Authenticate to Google Cloud'
+      uses: 'google-github-actions/auth@v1.0.0'
+      with:
+        workload_identity_provider: ${{ secrets.PROVIDER_NAME }}
+        service_account: ${{ secrets.SERVICE_ACCOUNT }}
+    - name: Set up Cloud SDK
+      uses: google-github-actions/setup-gcloud@v1
+    - name: Cloud Build
+      id: cloudbuild
+      env:
+        RELEASE_PROJECT: '${{ secrets.RELEASE_PROJECT }}'
+        BUCKET_NAME: '${{ secrets.RELEASE_BUCKET_NAME }}'
+      run: |
+        export VERSION=$(cat version.txt)
+        gcloud config set project $RELEASE_PROJECT
+        gcloud builds submit --config .release/upload_uberjars.yaml \
+          --substitutions=_VERSION="$VERSION",_BUCKET_NAME="$BUCKET_NAME"
+    - name: Cleanup Bucket
+      run: |
+        gsutil rm -f gs://"$BUCKET_NAME"/v"$VERSION"/*.json 2> /dev/null || true 
+    - name: Generate SHA
+      id: generate_sha
+      run: |
+        ./.release/generate_sha.sh
+        echo "RELEASE_TABLE=$(cat release_table.md)" >> $GITHUB_ENV
+    - name: Update Release Notes
+      id: update_release
+      uses: tubone24/update_release@v1.0
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        RELEASE_TAG: ${{ needs.release-please.outputs.releases_created }}
+      with:
+        body: ${{ env.RELEASE_TABLE }}
+  
+  snapshot-pr:
+    runs-on: ubuntu-latest
+    needs: upload_uberjars
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Set up JDK 17
+      uses: actions/setup-java@v3
+      with:
+        distribution: 'zulu'
+        java-version: '17'
+    - name: update version.txt
+      run: ./.release/snapshot.sh
+    - name: update pom.xml files
+      run: ./.release/update_versions.sh
+    - name: create pull request
+      uses: peter-evans/create-pull-request@v4
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+        title: 'chore: update to latest SNAPSHOT version'
+        body: |
+          Auto-generated by [create-pull-request][1]
+          [1]: https://github.com/peter-evans/create-pull-request
+        labels: |
+            tests:run
```

nit: label is `tests: run` with a space
```suggestion
            tests: run
```